docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<blockquote>We can find clealy the discord by Matrix profile. It indicate the discord in Matrix profile by far. </blockquote><p> </p><p>Maybe: In this case, the discord, indicated by matrix profile, is what we are looking for.</p><p> </p><p>What do you think? Because, any matrix profile with a unique global maxima can reveal a unique discord. Whether or not that discord is what we are looking for is a different story. Here, we use matrix profile to find discord, and we are lucky that the discovered discord is what we are looking for.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I agree with you</p>